src/metrics/include/metrics/metrics.hpp

```diff
@@ -119,6 +127,10 @@ class RegistryT final {
         return ClockSource::now();
     }
 
+    std::size_t getWorkerCount(const std::string& actorName, const std::string& opName) const {
```

Comment:
Wasn't sure if we wanted to account for current phase when figuring the worker count.

Reply:
I think for now we'll make a blanket statement that the workerCount is constant across phases for a given (actor,operation), but that may not always be the case. 

Let's add a comment saying basically...that 😄